storage/store_test.go

```diff
@@ -154,32 +153,6 @@ func TestStoreInitAndBootstrap(t *testing.T) {
 	if _, err := store.GetRange(1); err != nil {
 		t.Errorf("failure fetching 1st range: %s", err)
 	}
-
```

Comment:
I'm actually working on a new set of unit tests just for the status.  Since we don't get a status on start or split anymore, and only at the end of a scan, it's a little bit more involved.
I'm working on a PR for it right now.